pkgs/development/libraries/at-spi2-core/default.nix

```diff
@@ -30,7 +31,7 @@ stdenv.mkDerivation rec {
   outputs = [ "out" "dev" ];
 
   nativeBuildInputs = [ meson ninja pkg-config gobject-introspection makeWrapper ];
-  buildInputs = [ libX11 libXtst libXi ];
+  buildInputs = [ libX11 libXtst libXi libXext ];
```

Comment:
I wonder if this should instead go to `propagatedBuildInputs` of `libXi`. But weirdly, I do not see any propagation in the file: https://github.com/NixOS/nixpkgs/blob/c430ccbb77a263866d2b68e65a74ead595f55a76/pkgs/servers/x11/xorg/default.nix#L909

Maybe add a comment that it is required transitively by `libXi` for now.